packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1026,8 +1025,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return {
             // the defaults
             ... DefaultSummaryConfiguration,
-            // the server provided values
-            ... this.context?.serviceConfiguration?.summary,
```

Comment:
I thought that was already removed in previous PR, no?

Reply:
This is next.